### PR TITLE
Bulk validation update

### PR DESCRIPTION
**Pull request by jdiaz4302**
Essentially this is a (hopefully final) updating of validation set analysis.

Version controlling run with latitude and process-based estimates + having ran all comparisons + synthesis. More explicit comparison across model sizes. Separated out GLM vs GLM-free XAI vizualization notebooks.

The training code to produce these results simply changed the currently version-controlled setting of `remove_PB` to `False`

### CODE REVIEW

File: 03_eval/src/A_EvaluationMetrics.md [hunk 1]
```diff
@@ -689,11 +694,11 @@ np.savez_compressed(best_eval_metrics_fpath, **evals1)
...
-          'pred_error_ice_on':all_pred_ice_on_error[best_i],
-          'pred_error_ice_off':all_pred_ice_off_error[best_i],
-          'pred_error_dur':obs_dur - calc_ice_duration(all_pred_ice_off[best_i],
-                                                       all_pred_ice_on[best_i]),
-          'model_name_ls':eval_df['name'][best_i]}
+          'pred_error_ice_on':all_pred_ice_on_error[second_best_i],
+          'pred_error_ice_off':all_pred_ice_off_error[second_best_i],
+          'pred_error_dur':obs_dur - calc_ice_duration(all_pred_ice_off[second_best_i],
+                                                       all_pred_ice_on[second_best_i]),
+          'model_name_ls':eval_df['name'][second_best_i]}
```
Comment (jdiaz4302) [inline 03_eval/src/A_EvaluationMetrics.md:701]: this was also a typo/bug correction that had no impact on analyses because these metrics are never loaded after being saved
